docs/topics/testing/tools.txt

```diff
@@ -1706,7 +1706,17 @@ your test suite.
 
     .. versionchanged:: 3.2
 
-        Default value of ``transform`` changed from ``repr`` to ``None``
+        The default value of ``transform`` argument was changed to ``None``.
+
+    .. versionadded:: 3.2
+
+        Support for querysets comparison was added.
```

Comment:
```suggestion
        Support for direct comparison between querysets was added.
```

Strictly speaking this was always possible if you passed `transform=lambda x: x`, but was just not obvious. So does this really count as `versionadded` or just another part of `versionchanged`? Maybe we should say that the value was changed and this is now what it means.

Reply:
I think we can leave `versionadded`.